daxa/process/erosita/_common.py

```diff
@@ -19,6 +21,59 @@
 
 ALLOWED_EROSITA_MISSIONS = ['erosita_calpv']
 
+class _eSASS_Flag(Flag):
```

Comment:
I can make a commit to add a better doc string to the class if that works.
Might be helpful if in the future you want to replace it since, as it is, it's kind of overkill for just validating the flags.

Reply:
Ok lovely, I have made a co-authored commit with Toby on (i think that it was worked how it should), I put his full name and github username in the actual docstring, then attached his sussex email to the co-authored commit.